docs/rest/DaVinciPlanNet/SearchParameters/HealthcareServiceCategory.json

```diff
@@ -0,0 +1,51 @@
+{
+    "resourceType": "SearchParameter",
```

Comment:
do you need files in this folder?

Reply:
**This file is not part of the GraphQL work**, while doing a merge with the main branch, github took the new files from others as my changes.